test/functional/verify/verify-fix.bats

```diff
@@ -143,16 +143,18 @@ test_setup() {
 		.deleted
 		--picky removing extra files under .*/target-dir/usr
 		REMOVING /usr/untracked_file3
-		Inspected 19 files
+		REMOVING /usr/share/defaults/swupd/versionurl
```

Comment:
All tests with changes on this commit should be in the 1st commit of this series, right?

Reply:
> All tests with changes on this commit should be in the 1st commit of this series, right?

ooh you are right, I'll move those to the first commit